pyworkflow/em/protocol/protocol_micrographs.py

```
@@ -588,8 +588,11 @@ def _updateOutputCTFSet(self, micList, streamMode):
 
 
         for micFn, micDir, mic in self._iterMicrographs(micList):
-            ctf = self._createCtfModel(mic)
-            outputCtf.append(ctf)
+            try:
+                ctf = self._createCtfModel(mic)
```

Can you double-check that this not already handled at other level? As far as I remember the movies or micrographs protocols do not fails if one item fails. On the other side, generally  I don't like to hide errors without a good reason. ;)

---

The reason is to avoid the ctf protocol stops when one ctf estimation fails. It raise a message from the binary xmipp_error... I dont know what happen with ctffind and gctf (Pablo is going to check this) but we can put a message inside Scipion with some extra information ;)